Artifacts/windows-sharepoint2013B/SharePoint2013SP1P2.ps1

```diff
@@ -0,0 +1,52 @@
+$logFile = "InstallShpt2013SP1.log"
+$preInstaller = "prerequisiteinstaller.exe"
+$rebootTemp = "rebootTrue.log"
+
+$path = Join-path -path $env:ProgramData -childPath "DTLArt_Shpt2013"
+
+if(!(Test-Path -path $path)){
+    New-Item -ItemType Directory -Force -Path $path
+}
+
+$localFile = Join-Path $path $preInstaller
+$localLog = Join-Path $path $logFile
+$localreboot = Join-Path $path $rebootTemp
+
+If (Test-Path $localreboot){
+
+    Remove-Item -Path $localreboot
+
+    Add-content $localLog -value "Starting PreRequisite /continue process"
+
+    #Extract sharePoint Files from downloaded file
+
+    $argumentList = " /unattended /continue"
+
+    $retCode = Start-Process -FilePath $localFile -ArgumentList $argumentList -Wait -PassThru
+
+    if ($retCode.ExitCode -eq 0)
+    {
+        Add-content $localLog -value "SharePoint 2013 prerequisite installer 2 succeded - no reboot"

```

Comment:
If no reboot is required, by unconditionally make the call to reboot on line 52?